### ISSUE TITLE
[uflow] Question about training loss and training memory requirement

### ISSUE THREAD

**Issue by SwagJ** [google-research/google-research#615]
Hi @AustinCStone, 

Thank you for your great work. I just have a question about uflow's training loss and training memory requirement. 
1. Training loss: in your final SOTA result, the losses you used are self-sup + smoothness + occ masking + photo loss as census loss, right? But I just have one question: all these losses are applied only on model output level, i,e, level 2 in your implementation or the losses are applied on all levels?
2.  Since self-sup loss will engage after 50% of total iteration, how much memory will be required if using batch size of 1?

I am looking forward to your reply. Thank you very much!

Best,
